code/modules/mob/living/basic/lavaland/basilisk/basilisk.dm

```diff
@@ -0,0 +1,87 @@
+/// Watchers' ground-dwelling cousins, they shoot at you until they get into melee and absorb laser fire to power up.
+/mob/living/basic/mining/basilisk
+	name = "basilisk"
+	desc = "A territorial beast, covered in a diamond shell which absorbs heat. Its stare causes victims to freeze from the inside."
+	icon_state = "Basilisk"
+	icon_living = "Basilisk"
+	icon_dead = "Basilisk_dead"
```

Comment:
are you willing to change these icons to be more standardized (be fully lowercase) or should we leave that for a later day? your choice